frontend/src/views/Login.vue

```diff
@@ -119,17 +123,19 @@ export default {
     "login_success": "Vous êtes maintenant connecté",
     "login": "Connexion",
     "password": "Mot de passe",
-    "remember_me": "Se souvenir de moi"
+    "remember_me": "Se souvenir de moi",
+    "reset_password": "Mot de passe oublié ?"
   },
   "es": {
     "email": "Email",
     "error": "Error",
     "login_already": "Ya has iniciado sesión",
     "login_failure": "Error durante el inicio de sesión",
     "login_success": "Iniciaste sesión correctamente",
-    "login": "Iniciar sesión ",
+    "login": "Iniciar sesión",
     "password": "Contraseña",
-    "remember_me": "Recordarme"
+    "remember_me": "Recordarme",
+    "reset_password": "¿Se te olvidó tu contraseña?"
```

Comment:
@FranckKe I changed this translation from `Se te olvidó tu contraseña ?` to `¿Se te olvidó tu contraseña?`

Reply:
:+1: